docs/CODE_REVIEW.md

```diff
@@ -0,0 +1,145 @@
+# How to do code reviews for curl
+
+Anyone and everyone is encouraged and welcome to review code submissions in
+curl. This is a guide on what to check for and how to perform a successful
+code review.
+
+## All submissions should get reviewed
+
+All pull requests and patches submitted to the project should be reviewed by
+at least one experienced curl maintainer before that code is accepted and
+merged.
+
+## Let the tools and tests take the first rounds
+
+On initial pull requests, let the tools and tests do their job first and then
+start out by helping the submitter understand the test failures and tool
+alerts.
+
+## How to provide feedback to author
+
+Be nice. Ask questions. Provide examples or suggestions of improvements.
+Assume best intentions. Remember language barriers.
+
+All first-time contributors can become regulars. Let's help them go there.
+
+## Is this a change we want?
+
+If this is not a change that seems to be aligned with the project's path
+forward and as such cannot be accepted, inform the author about this sooner
+rather than later. Do it gently and explain why and possibly what could be
+done to make it more acceptable.
+
+## Code style
+
+Most code style nits are detected by checksrc but not all. Only leave remarks
+on style deviation once checksrc doesn't find anymore.
```

Comment:
```suggestion
on style deviation once checksrc doesn't find any more.
```

Reply:
I'm not sure I agree with this (but English is not my native language so I'm willing to get corrected on this). I intended the meaning as in "when it can't find problems any longer" which is typically how "anymore" differs from "any more" - as the latter is more about any further instances. One source on the difference: https://www.grammarly.com/blog/anymore-vs-any-more/

Reply:
It should be "any more" as in "any more errors"

Reply:
Thank you.